include/eve/arch/cpu/wide.hpp

```diff
@@ -129,7 +132,7 @@ inline namespace EVE_ABI_NAMESPACE
     //! Constructs a eve::wide by splatting a scalar value in all lanes
     template<scalar_value S>
     EVE_FORCEINLINE explicit  wide(S const& v)  noexcept
-                            : storage_base(detail::make(eve::as<wide>{}, v))
+                            : storage_base(detail::make(eve::as<wide>{}, static_cast<Type>(v)))
```

Comment:
Oh. Hmm. Oh.

THis means we ignore all checks. Hmm. Not sure it's good.

Reply:
What do you mean ?

Reply:
`static_cast<Type>` will force conversion from int to char for example.